rest-service/manager_rest/storage/storage_manager.py

```diff
@@ -83,6 +83,10 @@ def _sort_query(query, sort=None):
                 if order == 'desc':
                     column = column.desc()
                 query = query.order_by(column)
+        else:
+            default_sort = model_class.default_sort_column()
+            if default_sort:
```

Comment:
How can it be empty?

Reply:
the implementor of the model class might well do a `return None` if they want no default sort for that model